src/modules/directory.rs

```diff
@@ -683,14 +683,14 @@ mod tests {
 
     #[test]
     fn root_directory() {
-        let actual = ModuleRenderer::new("directory").path("/").collect();
-        #[cfg(not(target_os = "windows"))]
-        let expected = Some(format!(
-            "{}{} ",
-            Color::Cyan.bold().paint("/"),
-            Color::Red.normal().paint("🔒")
-        ));
-        #[cfg(target_os = "windows")]
+        let actual = ModuleRenderer::new("directory")
+            .config(toml::toml! {
+                [directory]
+                read_only = ""
+                read_only_style = ""
```

Comment:
Can we get a comment/note here with a bit of context? Something along these lines:
```rs
// NOTE: _disabled_ due to false positives on Windows
```